homeassistant/components/squeezebox/media_player.py

```diff
@@ -135,16 +131,18 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
         return
 
     _LOGGER.debug("Creating LMS object for %s", ipaddr)
-    lms = LogitechMediaServer(hass, host, port, username, password)
+    lms = Server(async_get_clientsession(hass), host, port, username, password)
+    known_servers.add(ipaddr)
```

Comment:
Getting the `ipaddr` with `socket.gethostbyname` does I/O. Please run that in the executor thread pool with `hass.async_add_executor_job`.

Reply:
Done. Added to new commit.